packages/editor/src/client/routes/pageEditor.tsx

```diff
@@ -160,6 +160,7 @@ export function PageEditor({id}: PageEditorProps) {
     }
   }, [pageData, hasChanged])
 
+  // Change this with notification ?
   useEffect(() => {
     const error = createError?.message ?? updateError?.message ?? publishError?.message
     if (error) Alert.error(error, 0)
```

Comment:
This is set with `Notification` in articleEditor. Should it be changed here too ?

Reply:
@camilleory no. In articleEdtior `Notification` should be replaced with `Alert`.